src/app/router.options.ts

```diff
@@ -1,18 +1,24 @@
 import type { RouterConfig } from '@nuxt/schema'
 
 export default <RouterConfig> {
-  // https://router.vuejs.org/api/interfaces/routeroptions.html#routes
-  // alternatively, could put this inside the setup for each page
   routes: _routes => [
     {
       name: RouteNameE.DASHBOARD,
-      path: '/:identifier',
+      path: '/:identifier?',
       component: () => import('~/pages/dashboard.vue').then(r => r.default || r),
       meta: {
         layout: 'business',
         title: 'Business Dashboard',
         breadcrumbs: [getBcrosHomeCrumb, getRegistryDashCrumb, getBusinessDashCrumb],
         staffBreadcrumbs: [getStaffDashCrumb, getBusinessDashCrumb]
+      },
+      beforeEnter(to, _from, next) {
+        // If identifier is missing
+        if (!to.params.identifier) {
+          next({ name: RouteNameE.DASHBOARD, params: { identifier: 'default_identifier' } })
```

Comment:
This shows an URL of `https://business-dashboard-dev--pr-167-4s1gnd8i.web.app/default_identifier`.

How about setting the default identifier to "XXX" so it looks more like an error in the URL? Or other ideas?